spec/lib/api/2.0/views-spec.js

```diff
@@ -0,0 +1,147 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Http.Api.Views', function () {
+    var viewsProtocol;
+    var _;
+
+    before('start HTTP server', function () {
+        this.timeout(10000);
+        viewsProtocol = {
+            load: sinon.stub(),
+            getAll: sinon.stub(),
+            get: sinon.stub(),
+            put: sinon.stub(),
+            unlink: sinon.stub()
+        };
+
+        return helper.startServer([
+            dihelper.simpleWrapper(viewsProtocol, 'Views')
+        ]).then(function() {
+            _ = helper.injector.get('_');
+        });
+    });
+
+    beforeEach('reset stubs', function () {
+        _(viewsProtocol).methods().forEach(function (method) {
```

Comment:
@yyscamper Personally, I prefer to do it this way because I think this technique can be understood by most developers without needing to reference sinon documentation.